specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2021-05-01-preview/kubernetesconfiguration.json

```diff
@@ -1,724 +0,0 @@
-{
-  "swagger": "2.0",
-  "info": {
```

Comment:
why remove this file ?

Reply:
There are no changes for `kubernetesconfiguration`, should I include it anyway?

Reply:
Do we know why we have this in first place as this is new API version and folder?

---
In reply to: [603394448](https://github.com/Azure/azure-rest-api-specs/pull/13562#discussion_r603394448) [](ancestors = 603394448)

Reply:
the branch for this API version was created based on an older version, therefore those files were present in this branch as well